doc/sources/installation/installation-windows.rst

```diff
@@ -230,6 +239,36 @@ kivy with git rather than a wheel there are some additional steps:
      export USE_GSTREAMER=1
 
    These variables must be set everytime you recompile kivy.
+   
+   **MSVC:**
```

Comment:
But I thought that's what your fix does. It uses sys.executable, which I assumed is changed for venv to find the right header files? Is that not the case?